Author: Daniel Kahn Gillmor <dkg@fifthhorseman.net>
Description: relax versions of dependencies

These versions are the ones in debian, and the octopus appears to
build and run fine against them.

See also
https://gitlab.com/sequoia-pgp/sequoia-octopus-librnp/-/merge_requests/109

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -61,8 +61,8 @@
 default-features = false
 
 [dependencies.rusqlite]
-version = "0.24"
-features = ["bundled"]
+version = "0.28"
+features = ["buildtime_bindgen"]
 
 [dependencies.sequoia-autocrypt]
 version = "0.24"
@@ -73,7 +73,7 @@
 default-features = false
 
 [dependencies.sequoia-net]
-version = "0.24"
+version = "0.25"
 optional = true
 default-features = false
 
@@ -104,10 +104,10 @@
 version = "1"
 optional = true
 [build-dependencies.libgit2-sys]
-version = "=0.11"
+version = "0.12"
 
 [build-dependencies.vergen]
-version = "5"
+version = "6"
 features = ["git"]
 default-features = false
 
